impeller/geometry/rect.h

```diff
@@ -151,6 +151,14 @@ struct TRect {
     return TRect(0.0, 0.0, size.width, size.height);
   }
 
+  template <class U, class FT = T>
```

Comment:
docstring would help here, it's not immediately clear to me what this is for.  it's copying a rect, but why the `enable_if_t`?  If this potentially throwing away precision implicitly?

Reply:
Done. It's only defined for float destinations. If you want to convert to an integer rect you have to use a method like RoundOut to specify the rounding behavior.